### PR TITLE
Added OpenEBS v1.3.0 manifest for AWS Marketplace Listing

### PR DESCRIPTION
**Pull request by muratkars**
- Added the manifest file using the scanned v1.3.0 images from the AWS registry

Signed-off-by: Murat Karslioglu <murat@mayadata.io>

### CODE REVIEW

**Comment by muratkars** [comment]
This PR addresses [2811](https://github.com/openebs/openebs/issues/2811)

File: docs/openebs-operator-1.3.0-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,739 @@
...
+        - name: OPENEBS_IO_JIVA_REPLICA_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/jiva:1.3-latest"
+        - name: OPENEBS_IO_JIVA_REPLICA_COUNT
+          value: "3"
+        - name: OPENEBS_IO_CSTOR_TARGET_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/cstor-istgt:1.3-latest"
+        - name: OPENEBS_IO_CSTOR_POOL_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/cstor-pool:1.3-latest"
+        - name: OPENEBS_IO_CSTOR_POOL_MGMT_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/cstor-pool-mgmt:1.3.0-latest"
```
Comment (muratkars) [inline docs/openebs-operator-1.3.0-aws.yaml]: Tag needs to be 1.3-latest not 1.3.0-latest

File: docs/openebs-operator-1.3.0-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,739 @@
...
+        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/m-exporter:1.3-latest"
+        - name: OPENEBS_IO_CSTOR_POOL_EXPORTER_IMAGE
+          value: "117940112483.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-2846528634/quay.io/openebs/m-exporter:1.3-latest"
+        # OPENEBS_IO_ENABLE_ANALYTICS if set to true sends anonymous usage
+        # events to Google Analytics
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
```
Comment (ChandanSagar) [inline docs/openebs-operator-1.3.0-aws.yaml]: Can we change this to "openebs-aws"?

File: docs/openebs-operator-1.3.0-aws.yaml [hunk 3]
```diff
@@ -0,0 +1,739 @@
...
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: OPENEBS_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-operator"
```
Comment (ChandanSagar) [inline docs/openebs-operator-1.3.0-aws.yaml]: Can we change this to "openebs-aws" as well?

File: docs/openebs-operator-1.3.0-aws.yaml (hunk 2 above)
Comment (muratkars) [reply docs/openebs-operator-1.3.0-aws.yaml]: Updated

File: docs/openebs-operator-1.3.0-aws.yaml (hunk 3 above)
Comment (muratkars) [reply docs/openebs-operator-1.3.0-aws.yaml]: Updated

**Pull request merged**